tutoraspects/templates/aspects/apps/aspects/migrations/alembic/versions/0014_add_course_names_fields.py

```diff
@@ -0,0 +1,122 @@
+"""
+add additional fields to course_names_dict and and course_names
+"""
+from alembic import op
+
+
+revision = "0014"
+down_revision = "0013"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute(
+        """
+        DROP TABLE IF EXISTS {{ ASPECTS_EVENT_SINK_DATABASE }}.course_names;
```

Comment:
I think these can be `CREATE OR REPLACE` instead of doing a drop / create?

Reply:
Ah good point. Do you think the dictionary statements should be `CREATE OR REPLACE` as well? I know you mentioned having some trouble with the dictionary statements. I haven't been able to reproduce that issue locally but I wanted to double check.

Reply:
Yeah, I think they should all be create or replace if it works for you.